main.py

```diff
@@ -52,19 +56,20 @@ def show_all_books():
 
 
 def search_book():
-    search_term = ui.ask_question('Enter search term, will match partial authors or titles.')
+    search_term = ui.ask_question(
```

Comment:
This is a minor change but it's unrelated to the specific issue.

Reply:
I noticed that when I click ctrl+s the spacing automatically changes for this one. Is there a way to update this in vscode?